test_scripts/API/Alert/007_ATF_Alert_UI_Alert_WARNINGS.lua

```diff
@@ -0,0 +1,162 @@
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+local resultCodes = {"SUCCESS", "WARNINGS", "WRONG_LANGUAGE", "RETRY", "SAVED"}
+for i=1,#resultCodes do
+  Test["TestStep_Alert_UI_Alert_WARNINGS_TTS_Speak_"..resultCodes[i]] = function(self)
+    local cor_id = self.mobileSession:SendRPC("Alert",
+    {
+      alertText1 = "alertText1",
+      alertText2 = "alertText2",
+      alertText3 = "alertText3",
+      ttsChunks = {{text = "TTSChunk", type = "TEXT"}},
+      duration = 3000,
+      progressIndicator = true,
+      softButtons =
+        {{
+          type = "BOTH",
+          text = "Close",
+          image = { value = "icon.png", imageType = "DYNAMIC"},
+          isHighlighted = true,
+          softButtonID = 3,
+          systemAction = "DEFAULT_ACTION",
+      }}
+    })
+
+    EXPECT_HMICALL("UI.Alert",
+    {alertStrings =
+      {
+        {fieldName = "alertText1", fieldText = "alertText1"},
+        {fieldName = "alertText2", fieldText = "alertText2"},
+        {fieldName = "alertText3", fieldText = "alertText3"}
+      },
+      alertType = "BOTH",
+      duration = 3000,
+      progressIndicator = true,
+      softButtons =
+        {{
+          type = "BOTH",
+          text = "Close",
+          image = { imageType = "DYNAMIC"},
+          isHighlighted = true,
+          softButtonID = 3,
+          systemAction = "DEFAULT_ACTION",
+      }},
+      appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+    })
+    :ValidIf(function(_,data)
+      local value_Icon = storagePath .. "icon.png"
+      if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
+        print("\27[31m value of menuIcon is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.cmdIcon.value .. "\27[0m")
+        return false
+      else
+        return true
+      end
+    end)
+    :Do(function(_,data2)
+      local function alertResponse()
+        self.hmiConnection:SendResponse(data2.id, "UI.Alert", "WARNINGS", { })
+      end
+      RUN_AFTER(alertResponse, 1500)
+    end)
+
+    EXPECT_HMICALL("TTS.Speak",
+    {ttsChunks =
+        {{
+          text = "TTSChunk",
+          type = "TEXT"
+      }},
+      speakType = "ALERT",
+      appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+    })
+    :Do(function(_,data3)
+      self.hmiConnection:SendNotification("TTS.Started",{ })
+      local function ttsSpeakResponse()
+        self.hmiConnection:SendResponse (data3.id, data3.method, resultCodes[i], {})
+      end
+      RUN_AFTER(ttsSpeakResponse, 1000)
+    end)
```

Comment:
@vvvakulenko, 
suggestion: will be more readable if you use one empty row before EXPECT_RESPONSE